source/features/github-bugs.css

```diff
@@ -51,10 +51,14 @@
 /* Mute unclickable reaction buttons, disable hover */
 /* Test: https://github.com/refined-github/sandbox/pull/48 */
 .social-reaction-summary-item[disabled] {
-	pointer-events: none;
+	pointer-events: auto;
 	filter: grayscale(0.5);
 }
 
+.social-reaction-summary-item:hover[disabled] {
```

Comment:
`:hover` is not necessary I think, it can just be part of the main rule. If the selector is not strong enough, we can use `:root .social-reaction-summary-item[disabled]` for the whole rule.

Reply:
Indeed it was. There was no selector weakness issue too.